app/uploaders/portrait_uploader.rb

```diff
@@ -0,0 +1,51 @@
+# encoding: utf-8
+
+class PortraitUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
```

Comment:
Are we going to set this up with an s3 bucket? I think if there are enough users or just people in the app it might lag.